projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.ts

```diff
@@ -31,8 +31,8 @@ export class BreadcrumbComponent implements OnInit {
 
   private setTitle(): void {
     this.title$ = this.pageMetaService.getMeta().pipe(
-      filter(Boolean),
-      map((meta: PageMeta) => meta.heading || meta.title)
+      filter(isNonNullable),
+      map((meta) => meta.heading ?? meta.title ?? '')
```

Comment:
We're changing the logic a bit. Deliberately? Previously when `heading` was an empty string, we did fallback to `title`. Now we fallback to title only when `heading` is Nullable